src/utils/urls.js

```diff
@@ -0,0 +1,14 @@
+export const getLoginUrl = () => (
+  `${process.env.REACT_APP_EGGHEAD_BASE_URL}/users/jwt?return_to=${window.location.href}`
+)
+
+export const guideUrl = 'https://instructor.egghead.io/guide'
+
+export const chatUrl = 'https://eggheadio.slack.com/messages/egghead-instructors/'
```

Comment:
@tayiorbeii That's a good point. Ideally, I'd like one of them to open up a new chat with Joel, John, and I. I'll see if this is possible with a URL structure or some sort of handler.